### DISCUSSION TITLE
HTML to YAML conversion

### DISCUSSION THREAD

**Discussion by MarianoCh** [statamic/cms#7002] [Q&A]
Hello, I am quite a beginner and only involved in a larger project that is a bit stuck at one point. Many of our texts are still created in Word or other editors. We then get them into HTML to some extent, but the path to YAML is not yet made. Are there already any adaptable converters for this? Or are there already any templates for Pandoc that we could adapt? But maybe this question is already a lot of nonsense... Thank you!

**Comment by edalzell** [comment]
What do you mean "path to YAML"?

Can you give an example of what you'd like to do?

**Reply by MarianoCh** [reply]
Hi Erin,
We finally found someone who adopted the "html-to-prosemirror" to convert Word resp. the Word-HTML-Output of our documents to prosemirror. Sorry for not letting everyone know right away.
Thank you!
